# ロボットバージョンの名称GET
---
string name

